packages/ffe-lists/less/regular-lists.less

```diff
@@ -52,7 +52,7 @@
 .ffe-bullet-list {
     padding: 0;
     text-align: left;
-    margin: 0 0 20px;
+    margin: -10px 0 20px;
```

Comment:
Instead of inverting the margin here, how about just removing the padding from the first list item?

In other words:

```css
> li {
  margin-top: 10px;
  &:first-of-type {
    margin-top: 0;
  } 
  // ...
}
```

Both obviously work, but it's easier (at least to me) to comprehend how the `:first-of-type` selector works, than negative margins.